includes/class-lkn-payment-banking-slip-pix-for-lifterlms-helper.php

```diff
@@ -127,4 +126,71 @@ final public static function dependency_alert(): void {
     final public static function inactive_alert(): void {
         add_action('admin_notices', array('Lkn_Payment_Banking_Slip_Pix_For_Lifterlms_Helper', 'inactive_notice'));
     }
+
+    /**
+     * Array for pick the data of the gateways settings in LifterLMS.
+     *
+     * @since 1.0.0
+     *
+     * @param string $gateway_id
+     * @return array $configs
+     */
+    final public static function get_configs($gateway_id) {
+        $configs = array();
+
+        $configs['logEnabled'] = get_option(sprintf('llms_gateway_%s_logging_enabled', $gateway_id), 'no');
+        $configs['baseLog'] = LKN_PAYMENT_BANKING_SLIP_PIX_FOR_LIFTERLMS_DIR . 'includes/logs/' . date('d.m.Y-H.i.s') . '.log';
+
+        $configs['paymentInstruction'] = get_option(sprintf('llms_gateway_%s_payment_instructions', $gateway_id), __('Check the payment area below.', 'payment-banking-slip-pix-for-lifterlms'));
+        $configs['apiKey'] = get_option(sprintf('llms_gateway_%s_api_key', $gateway_id));
+        $configs['tokenKey'] = get_option(sprintf('llms_gateway_%s_token_key', $gateway_id));
+        $configs['daysDueDate'] = get_option(sprintf('llms_gateway_%s_days_due_date', $gateway_id));
+
+        $configs['urlPix'] = 'https://pix.paghiper.com/';
+        $configs['urlSlip'] = 'https://api.paghiper.com/';
+
+        return $configs;
+    }
+
+    /**
+     * Catch the url of the current page.
+     *
+     * @since 1.0.0
+     *
+     * @return string $page_url
+     */
+    final public static function get_current_url() {
+        $page_url = 'http';
+        if ('on' === $_SERVER['HTTPS']) {
+            $page_url .= 's';
+        }
+        $page_url .= '://';
+        if ('80' !== $_SERVER['SERVER_PORT']) {
+            $page_url .= $_SERVER['SERVER_NAME'] . ':' . $_SERVER['SERVER_PORT'] . $_SERVER['REQUEST_URI'];
+        } else {
+            $page_url .= $_SERVER['SERVER_NAME'] . $_SERVER['REQUEST_URI'];
+        }
+
+        return $page_url;
```

Comment:
@FabricioLinkNacional Needs sanitization, please use [this function](https://developer.wordpress.org/reference/functions/sanitize_url/) to sanitize the $page_url